excalidraw-app/data/LocalData.ts

```diff
@@ -43,6 +43,12 @@ import { FileManager } from "./FileManager";
 import { Locker } from "./Locker";
 import { updateBrowserStateVersion } from "./tabSync";
 
+// Global state to inform the app about local storage overflow
+import { atom } from "jotai";
+import { appJotaiStore } from "../app-jotai";
+
+export const isLocalStorageOverflowAtom = atom(false);
```

Comment:
Since this is already exported, you might not need the `appJotaiStore` and just set & use the atom directly.

Reply:
I tried `useSetAtom`, but it expects to be called within a component or a custom hook. Is there another way to set atoms directly outside of component and hook context?